packages/ripple-tide-grant/index.js

```diff
@@ -0,0 +1,89 @@
+import mime from 'mime-types'
+import { getField, getLinkFromField } from '@dpc-sdp/ripple-tide-api/src/services/utils'
+import components from './component-loader'
+
+// Utils
+const formatAudiences = (audiences) => {
+  const formatAudience = (input) => {
+    const term = typeof input === 'string' ? input : input.name
+    if (term) {
+      switch (term) {
+        case 'Individual':
+          return 'individuals'
+        case 'Business':
+          return 'businesses'
+        default:
+          return term.toLowerCase()
+      }
+    }
+  }
+  if (audiences && audiences.length > 0) {
+    const audienceStr = [...new Set(audiences)].map(a => formatAudience(a)).join(', ')
+    return `${audienceStr.charAt(0).toUpperCase() + audienceStr.slice(1)}`
+  }
+  return ''
+}
+
+const formattedSize = (fileSize) => {
+  if (fileSize != null) {
+    // https://stackoverflow.com/a/18650828
+    if (typeof fileSize === 'string') return fileSize
+    if (fileSize === 0) return '0 Bytes'
+    const k = 1024
+    const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
+    const i = Math.floor(Math.log(fileSize) / Math.log(k))
+    return parseFloat((fileSize / Math.pow(k, i)).toFixed(2)) + ' ' + sizes[i]
+  }
+  return ''
+}
+
+export default {
+  pageComponent: () => import(/* webpackMode: "eager" */ '@dpc-sdp/ripple-tide-grant/index.vue'),
+  components,
+  mapping: {
+    title: 'title',
+    overview: (src) => ({
```

Comment:
I've added support for object syntax now so you should be able to refactor this to just use the string path in these nested fields